core/src/main/java/com/klaytn/caver/account/AccountKeyFail.java

```diff
@@ -64,4 +71,20 @@ public String getRLPEncoding() {
     public static String getType() {
         return TYPE;
     }
+
+    public static class AccountKeyFailSerializer extends JsonSerializer<AccountKeyFail> {
```

Comment:
Can you add simple description for serialize / deserialize functions?

Reply:
I added it